vernac/vernacentries.ml

```diff
@@ -1428,10 +1466,22 @@ let vernac_arguments ~section_local reference args more_implicits nargs_for_red
               strbrk "are relevant for constants only.")
   end;
 
+  if bidi_hint_specified then begin
+    let n = Option.get nargs_before_bidi in
+    if section_local then
+      Pretyping.add_bidirectionality_hint sr n
+    else
+      Lib.add_anonymous_leaf (inBidiHints (sr, n))
+    end;
+
+  if clear_bidi_hint then
+    Pretyping.clear_bidirectionality_hint sr;
```

Comment:
What about when we're not `section_local`?

Reply:
Oh, right, I completely forgot to also port this. Will do, thanks!

Reply:
Done.